packages/acs-ui-common/package.json

```diff
@@ -7,7 +7,7 @@
   "types": "dist/acs-ui-common.d.ts",
   "scripts": {
     "build": "rushx build:esm && rushx build:cjs && api-extractor run --local",
-    "build:esm": "tsc",
+    "build:esm": "rimraf dist/dist-esm && tsc",
```

Comment:
Rush build detects if changes are made in the repo since last build, can you test running something like `rush build -t @azure/communication-react` twice and ensure the project and dependent packlets aren't rebuilt

Reply:
Yep:

```
codespace ➜ /workspaces/communication-ui-library/packages/storybook (prprabhu/really-clean-build) $ rush build -t communication-reactFound configuration in /workspaces/communication-ui-library/rush.json


Rush Multi-Project Build Tool 5.47.0 - https://rushjs.io
Node.js version is 14.15.1 (LTS)

Found configuration in /workspaces/communication-ui-library/rush.json

Starting "rush build"

Executing a maximum of 4 simultaneous processes...

==[ @internal/acs-ui-common ]======================================[ 1 of 8 ]==
@internal/acs-ui-common was skipped.

==[ @internal/calling-stateful-client ]============================[ 2 of 8 ]==
@internal/calling-stateful-client was skipped.

==[ @internal/chat-stateful-client ]===============================[ 3 of 8 ]==
@internal/chat-stateful-client was skipped.

==[ @internal/react-components ]===================================[ 4 of 8 ]==
@internal/react-components was skipped.

==[ @internal/calling-component-bindings ]=========================[ 5 of 8 ]==
@internal/calling-component-bindings was skipped.

==[ @internal/chat-component-bindings ]============================[ 6 of 8 ]==
@internal/chat-component-bindings was skipped.

==[ @internal/react-composites ]===================================[ 7 of 8 ]==
@internal/react-composites was skipped.

==[ @azure/communication-react ]===================================[ 8 of 8 ]==
@azure/communication-react was skipped.



==[ SKIPPED: 8 projects ]======================================================

These projects were already up to date:
  @azure/communication-react
  @internal/acs-ui-common
  @internal/calling-component-bindings
  @internal/calling-stateful-client
  @internal/chat-component-bindings
  @internal/chat-stateful-client
  @internal/react-components
  @internal/react-composites


rush build (0.11 seconds)
```

This is expected because rush tracks changes to _source files_, and we're only cleaning the build folder here.

Reply:
Oh nice, how does rush determine what are source files and what is the build folder?

Reply:
Ah cool, (if I'm reading this right) it appears to store a store of file hashes of git tracked files: https://rushjs.io/pages/advanced/incremental_builds/

Reply:
Ah you can see what files are being tracked too, I wonder if there's flexibility to remove files from this list - I would like to prevent rebuilds happening when a new api.md file is generated:
![image](https://user-images.githubusercontent.com/2684369/133317517-de6ea93f-c2cf-49df-97d6-f6eb61e507b1.png)

(Currently if you `build` twice but the first updated the api.md the second `build` does a full rebuild of that packlet)